Cards/Resource/Card_Energy.tres

```diff
@@ -14,14 +14,14 @@ effect = SubResource("Resource_jur0i")
 value = 70
 target_type = 0
 
-[sub_resource type="Resource" id="Resource_1100t"]
-script = ExtResource("3_2mfb5")
-energy_cost = 0
```

Comment:
so the card that gives you energy costs 2 energy ?

Reply:
Can you rephrase this?

Reply:
Nvm I saw the card, it gives you 70 energy for a cost of 2
I was just surprised that the card that adds to your energy costs energy, but since it gives a lot it doesn't matter
this will be changed later to a more reasonable value